config/locales/idv/fr.yml

```diff
@@ -85,16 +85,39 @@ fr:
           problème persiste, revenez plus tard.
         text_html: Veuillez réessayer. Si vous continuez à recevoir ces erreurs, %{link}
       phone:
-        fail_html: '<strong>Veuillez réessayer dans %{timeout}.</strong> Pour votre
-          sécurité, nous limitons le nombre de fois où vous pouvez tenter de
-          vérifier votre numéro de téléphone en ligne.'
         heading: Nous n’avons pas pu faire correspondre ce numéro de téléphone à
           d’autres enregistrements
         jobfail: Un problème s’est produit et nous ne pouvons pas traiter votre demande
           pour le moment. Veuillez réessayer.
+        rate_limited:
+          body: Pour des raisons de sécurité, nous limitons le nombre de tentatives de
+            vérification de votre numéro de téléphone en ligne. Vous devrez
+            attendre %{time_left} et recommencer avant d’essayer à nouveau de
+            vérifier votre identité par téléphone.
+          gpo:
+            button: Vérifier par courrier
+            prompt: Continuez maintenant en vérifiant par courrier, ce qui prend
+              généralement de 3 à 7 jours ouvrables.
+          heading: Nous n’avons pas pu vérifier votre identité par téléphone
         timeout: Notre demande de vérification de vos renseignements a expiré. Veuillez
           réessayer.
-        warning: Veuillez vérifier les informations que vous avez saisies et réessayer.
+        warning:
+          attempts:
+            one: Pour des raisons de sécurité, il vous reste une tentative.
+            other: Pour des raisons de sécurité, il vous reste %{count} tentatives.
+          gpo:
+            button: Vérifier par courrier
+            explanation: Si vous n’avez pas d’autre numéro de téléphone à essayer, vérifiez
+              plutôt par courrier.
+            heading: Vérifier par courrier
+            how_long_it_takes: Cette procédure prend typiquement 3 à 7 jours ouvrables.
+          heading: Nous n’avons pas pu vous associer à ce numéro
+          learn_more_link: 'Más información sobre qué número de teléfono usar'
```

Comment:
This is Spanish in the French translation file.

Reply:
Doh! Fixed in 0ce5bb44cd4a4adaf6f50af421f60305b5a0976f